src/streamlink/plugins/niconicochannelplus.py

```diff
@@ -0,0 +1,67 @@
+"""
+$description NicoNico Channel Plus (nicochannel+) is a new feature of Niconico Channel.
+$url nicochannel.jp
+$type vod
+$account Not required. Full-length videos are accessible.
+$notes Downloading non-free content is not recommended.
```

Comment:
That info would be unneeded if we'd merge this. For custom plugins though that get side-loaded, no metadata at all is needed.

Reply:
I agree.
They are not informative. Removed in c9654cc1f672b8eb36b1da9896485202bbceea97 .